rpcs3/rpcs3_app.cpp

```diff
@@ -168,36 +170,64 @@ void rpcs3_app::InitializeCallbacks()
 
 		bool disableMouse = guiSettings->GetValue(gui::gs_disableMouse).toBool();
 
+		// Make display geometry rectangle for the gs frame
+		// Take area of main window & center it neatly inside of it
+		// This prevents the frame being spawned wherever your cursor is,
+		// which is really annoying for multi-screen users
+		QRect main_window_geometry = RPCS3MainWin->geometry();
+
+		// Get minimum virtual screen x & y for clamping the
+		// window x & y later while taking the width and height
+		// into account, so they don't go offscreen
+		int min_screen_x = std::numeric_limits<int>::max();
+		int max_screen_x = std::numeric_limits<int>::min();
+		int min_screen_y = std::numeric_limits<int>::max();
+		int max_screen_y = std::numeric_limits<int>::min();
+		foreach(auto screen, QApplication::screens())
+		{
+			min_screen_x = std::min(min_screen_x, screen->geometry().x());
```

Comment:
maybe use availableGeometry to exclude the taskbar as well

Reply:
good call, I'll add it in a bit